web/modules/custom/solution/src/SolutionAffiliationFieldItemList.php

```diff
@@ -169,4 +173,111 @@ protected function solutionInOfficialGraph(): bool {
     return $this->solutionInOfficialGraph;
   }
 
+  /**
+   * Returns the graph ids that the collection exists in.
+   *
+   * @return array
+   *   An array of graph URIs that the collection exists in indexed by graph id.
+   */
+  protected function getAvailableGraphs(): array {
+    $graph_handler = \Drupal::service('sparql.graph_handler');
+    $return = [];
+
+    foreach (['default', 'draft'] as $graph_id) {
```

Comment:
We should not hardcode the "official" graph list. Graph handler has dedicated method.